pkg/virt-handler/vm_test.go

```diff
@@ -1286,6 +1288,24 @@ func (m *MockGracefulShutdown) TriggerShutdown(vmi *v1.VirtualMachineInstance) {
 	Expect(err).NotTo(HaveOccurred())
 }
 
+type MockIsolationDetector struct{}
```

Comment:
This should be auto generated?

Reply:
OK I've found a way to make it work without manually writing a fake.